src/main/java/nextstep/subway/station/domain/Station.java

```diff
@@ -15,15 +16,24 @@
     public Station() {
     }
 
-    public Station(String name) {
+    Station(Long id, String name) {
+        this.id = id;
         this.name = name;
     }
 
+    public Station(String name) {
+        this(null, name);
```

Comment:
아래와 같이 정의해도 동일한 결과를 얻을 수 있지 않을까요?

제가 혹시 의도를 놓친게 있다면 코멘트 남겨주세요!

```suggestion
        this.name = name;
```

Reply:
사실 주생성자와 보조 생성자를 표현하고 싶었습니다 ㅎㅎ
이름밖에 없는 엔티티라 미묘하긴 하네요 ㅜ 차라리 '정적 팩토리 메서드가 나았나' 하는 고민도 듭니다